specification/policyinsights/resource-manager/Microsoft.PolicyInsights/stable/2019-10-01/policyStates.json

```diff
@@ -495,6 +495,83 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.PolicyInsights/policyStates/latest/triggerEvaluation": {
+      "post": {
+        "operationId": "PolicyStates_TriggerSubscriptionEvaluation",
+        "description": "Triggers a policy evaluation scan for all the resources under the subscription",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The scan is done."
+          },
```

Comment:
do we get 200 or is this return code from the operation pointed by the location?

Reply:
The operation pointed by the location.  The validation tools require a terminal status code be defined.